### ISSUE TITLE
Support for Taurus

### ISSUE THREAD

**Issue by mmosallem** [microsoft/azure-load-testing#99]
**Is your feature request related to a problem? Please describe.**
Authoring jmeter test scripts depends heavily on using the jmeter GUI, many users adopted Taurus as a more friendly way of authoring jmeter tests using yaml. Currently Azure Load Testing supports running jmeter scripts (jmx files) only. We cannot re-use our existing testing scenarios (writing using taurus yml files) 

**Describe the solution you'd like**
Add a third option to create a test by uploading a taurus yaml file.

**Describe alternatives you've considered**

**Additional context**

**Comment by mmosallem** [comment]
@Sachid26 Yes we have existing taurus yaml scripts that we would like to upload to Azure Load Testing and run them there as we would upload jmx files.

**Comment by Sachid26** [comment]
@mmosallem ... We can add this to our backlog, but there is no ETA i can provide at this time.
